### PR TITLE
ML-041/add associations specs

### PR DESCRIPTION
**Pull request by WhatWasThatt**
# Description

config shoulda matchers
config rubocop.yml for use should
add associations specs
bring tests to the same style guide

## Trello card link

https://trello.com/c/G0idcjaX

# Checklist:

- [x] My code fixes the problem or adds required functionality
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove that my feature works
- [x] New and existing unit tests pass locally with my changes

# Additional requirements:

- [ ] Does your code require any migrations to be ran?
- [ ] Does your code require installing any new gems or system dependencies?
- [ ] Does your code require any changes to current deployment process?

# Additional Information

### CODE REVIEW

**Review by KarinaKlevzhits — DISMISSED** [review]

**Review by KarinaKlevzhits — APPROVED** [review]

File: spec/routing/api/v1/ratings_routing_spec.rb [hunk 1]
```diff
@@ -1,29 +1,33 @@
 require 'rails_helper'
 
-RSpec.describe Api::V1::RatingsController, type: :routing do
-  describe 'routing' do
-    it 'routes to #index' do
-      expect(get: '/api/v1/ratings').to route_to('api/v1/ratings#index')
-    end
+module Api
```
Comment (ShrewHools) [inline spec/routing/api/v1/ratings_routing_spec.rb:3]: what is the reason to wrap in modules?

Comment (WhatWasThatt) [reply spec/routing/api/v1/ratings_routing_spec.rb:3]: it turned out to be useless

Comment (ShrewHools) [reply spec/routing/api/v1/ratings_routing_spec.rb]: Module wraps are great places to have controllers, services, concerns, constants and any other code that, by having the same responsibility they should stay together)
namespacing is power) where it is of course really required)

**Review by ShrewHools — APPROVED** [review]
GJ

**Review by KarinaKlevzhits — APPROVED** [review]

File: spec/routing/api/v1/ratings_routing_spec.rb (hunk 1 above)
Comment (WhatWasThatt) [reply spec/routing/api/v1/ratings_routing_spec.rb]: thanks)

**Pull request merged**